TouristEquipmentRental/TouristEquipmentRental.cpp

```diff
@@ -1,37 +1,73 @@
 #include <iostream>
+#include <cwchar>
 #include "FileDatabase.h"
+#include "PrimmaryView.h"
+#include "UserPanel.h"
 #include "DatabaseOperator.h"
+#include "GlobalOperator.h"
+#include "SignInUpView.h"
 
-using namespace std;
 
-int main()
-{
+
+int main(int argc, char** argv)
+{	
+	GlobalOperator style;
+	PrimmaryView Start;
+	SignInUpView Login, Register;
+
+	UserPanel UserPanel;
+
     auto fileDatabase = new FileDatabase();
 
-    auto databaseOperator = new DatabaseOperator(* fileDatabase);
+	auto databaseOperator = new DatabaseOperator(*fileDatabase);
+	
+	style.SetFont(24,0);
+
+	
+	//for (int i = 0; i < 255; i++) {
+	//	style.SetColor(i);
+	//	cout << "KOLOR NUMER " << i << endl;
+	//}
+
+	Start.RenderStartView();
+		system("cls");
+		Login.RenderLoginView();
+		/*Sgin In*/
+		try
+		{
+			auto user = databaseOperator->GetUser(Login.GetEmail(), Login.GetPassword());
+			if ((user.Id()[0] == 'A') && (user.Activated() == true)) {
+				style.Delay(1.5);
+				cout << "Zalogowano pomyslnie do panelu administratora" << endl;
+				cout << endl;
+				system("cls");
+				//AdminPanel.RenderAdminMenu();
+			} 
+			else {
```

Comment:
This should check if the user account is activated too, otherwise anyone in the database can log in

Reply:
Or maybe we should delete checking if user is activated, because there is logic about displaying that in UserPanel